src/pages/Chart/paywallBoundaries.js

```diff
@@ -1,40 +1,59 @@
 import { getTimeIntervalFromToday, DAY, MONTH } from '../../utils/dates'
 
-const options = { isUTC: true }
+export function createOptions (hours = 48) {
+  const NEXT_DAY = new Date()
+  NEXT_DAY.setHours(hours, 0, 0, 0)
+  return { isUTC: true, from: NEXT_DAY }
+}
+
+const { from: ANON_LEFT_BOUNDARY_DATE } = getTimeIntervalFromToday(
+  -3,
+  MONTH,
+  createOptions(72)
+)
+const { from: ANON_RIGHT_BOUNDARY_DATE } = getTimeIntervalFromToday(
+  -1,
+  DAY,
+  createOptions(24)
+)
 
 const { from: FREE_LEFT_BOUNDARY_DATE } = getTimeIntervalFromToday(
   -24,
   MONTH,
-  options
+  createOptions()
 )
 const { from: FREE_RIGHT_BOUNDARY_DATE } = getTimeIntervalFromToday(
   -1,
   MONTH,
-  options
+  createOptions()
 )
 
 const { from: BASIC_LEFT_BOUNDARY_DATE } = getTimeIntervalFromToday(
   -24,
   MONTH,
-  options
+  createOptions()
 )
 const { from: BASIC_RIGHT_BOUNDARY_DATE } = getTimeIntervalFromToday(
   -7,
   DAY,
-  options
+  createOptions()
 )
 
 const { from: PRO_LEFT_BOUNDARY_DATE } = getTimeIntervalFromToday(
   -60,
   MONTH,
-  options
+  createOptions()
 )
 const PRO_RIGHT_BOUNDARY_DATE = false
 
 const ENTERPRISE_LEFT_BOUNDARY_DATE = false
 const ENTERPRISE_RIGHT_BOUNDARY_DATE = false
 
 export default {
+  ANON: {
```

Comment:
@DmitriVanGuard Why do we need to precalculate boundaries for anon and rest (all) plans? I dont' check full file, see only a part.

Reply:
What is the alternative?